dvc/config.py

```diff
@@ -285,8 +284,8 @@ def get_global_config_dir():
         """
         from appdirs import user_config_dir
 
-        return user_config_dir(
-            appname=Config.APPNAME, appauthor=Config.APPAUTHOR
+        return pathlib.Path(
```

Comment:
I'm not quite sure this pathlib stuff is really needed in this PR. It makes you solve additional issues with fspath, but I'm still not sure if you really need this at all.